src/main/java/com/synopsys/integration/jenkins/detect/extensions/DetectDownloadStrategy.java

```diff
@@ -22,10 +22,16 @@
  */
 package com.synopsys.integration.jenkins.detect.extensions;
 
+import java.io.Serializable;
+
 import hudson.model.AbstractDescribableImpl;
 import hudson.model.Descriptor;
 
-public abstract class DetectDownloadStrategy extends AbstractDescribableImpl<DetectDownloadStrategy> {
+public abstract class DetectDownloadStrategy extends AbstractDescribableImpl<DetectDownloadStrategy> implements Serializable {
+    private static final long serialVersionUID = 8059287675134866543L;
+
+    public abstract String getDisplayName();
```

Comment:
Any particular reason why we added this method instead of just doing DownloadStrategy.getDescriptor().getDisplayName()?  I have no objection to this, since it's potentially cleaner, but I just wonder if we're maintaining more lines of code than we need to.

Reply:
This is what I originally had done, until I started writing tests. downloadStrategy.getDescriptor().getDisplayName() throws a null pointer in my local environment so I decided to just implement the getter.

Reply:
In reviewing this edit, and ignoring the tests, I do think adding this getter is clearer/cleaner.

Reply:
Sounds good, let's keep it.